src/components/InvitationInfoModal/index.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import InfoIcon from "@mui/icons-material/Info";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Fade from "@mui/material/Fade";
+import Modal from "@mui/material/Modal";
+import Typography from "@mui/material/Typography";
+import { useState } from "react";
+
+import { EnrollmentForm as EnrollmentFormType } from "@/types/EnrollmentForm";
```

Comment:
I think it's fine to just import this as `import { EnrollmentForm } from "@/types"`